lib/superagent-mock.js

```diff
@@ -95,7 +95,23 @@ function mock (superagent, config) {
         var fixtures = parser.fixtures(match, this.params, this.headers);
         fn(null, parsers[this.url].callback(match, fixtures));
       } catch(err) {
-        fn(err, undefined);
+        var response = new superagent.Response({
```

Comment:
Sadly enough they are. I only added what was fully needed to get it to pass, but the Response constructor tries to get at a bunch of properties.

There might be a neater way to do this.. to somehow simulate a real response?